fastly/fixtures/ftps/get.yaml

```diff
@@ -1,30 +1,32 @@
 ---
 version: 1
-rwmutex: {}
 interactions:
 - request:
     body: ""
     form: {}
     headers:
-      Fastly-Key:
```

Comment:
@ezkl @phamann not sure why this got removed.

Reply:
We now have a filter which removes all `Fastly-Key` headers from VCR recordings. See: https://github.com/fastly/go-fastly/blob/master/fastly/fastly_test.go#L61-L65 

These files would have been produced before that change.